Makefile

```diff
@@ -0,0 +1,37 @@
+SHELL=/bin/bash
+.PHONY: build-ngsa-app create delete
+
+all: create deploy-ngsa-app
+
+delete:
+	@k3d cluster delete ngsa-app
+
+create: delete
+	@k3d cluster create ngsa-app --registry-use k3d-registry.localhost:5000 --config deploy/k3d.yaml --k3s-arg "--no-deploy=traefik@server:0"
+
+	@kubectl wait node --for condition=ready --all --timeout=60s
+	@sleep 5
+	@kubectl wait pod -A --all --for condition=ready --timeout=60s
+
+	@istioctl install -y --set profile=demo -f deploy/istio-operator.yaml
+
+	@kubectl create namespace ngsa
+
+	@kubectl label namespace ngsa istio-injection=enabled --overwrite
+
+build-ngsa-app:
+	docker build . -t localhost:5000/ngsa-app:local
+	docker push localhost:5000/ngsa-app:local
+
+deploy-ngsa-app: build-ngsa-app
```

Comment:
Add separate target for ngsa-memory and ngsa-cosmos.
Make the ngsa-memory default